src/System Application/App/AI/src/Copilot/CopilotCapabilityImpl.Codeunit.al

```diff
@@ -442,6 +443,28 @@ codeunit 7774 "Copilot Capability Impl"
             GuidedExperience.ResetAssistedSetup(ObjectType::Page, Page::"Copilot AI Capabilities");
     end;
 
+    procedure DeactivateCapability(var CopilotSettingsLocal: Record "Copilot Settings")
+    var
+        CopilotDeactivate: Page "Copilot Deactivate Capability";
+        ALCopilotFunctions: DotNet ALCopilotFunctions;
+    begin
+        if ALCopilotFunctions.IsCopilotFeedbackEnabled() then begin
+            CopilotDeactivate.SetCaption(Format(CopilotSettingsLocal.Capability));
+            if CopilotDeactivate.RunModal() = Action::OK then begin
+                CopilotSettingsLocal.Status := CopilotSettingsLocal.Status::Inactive;
+                CopilotSettingsLocal.Modify(true);
+
+                SendDeactivateTelemetry(CopilotSettingsLocal.Capability, CopilotSettingsLocal."App Id", CopilotDeactivate.GetReason());
```

Comment:
Can we just send this irregardless of the feedback?
The reason being empty is fine, but we still want the telemetry that they were deactivated

Reply:
Good catch. When feedback is disabled, we send telemetry but without reason